src/test/util/setup_common.cpp

```diff
@@ -168,8 +167,8 @@ ChainTestingSetup::~ChainTestingSetup()
     m_node.mempool.reset();
     m_node.scheduler.reset();
     m_node.chainman->Reset();
+    m_node.chainman->m_blockman.m_block_tree_db.reset();
```

Comment:
Same question here: since chainman owns the blockman which owns the pblocktree, perhaps the `m_node.chainman.reset()` is enough?

Reply:
Good point, done